--- httpd.conf	2009-05-01 08:13:50.032989000 +0800
+++ httpd.conf	2009-12-03 23:11:13.660039900 +0800
@@ -288,7 +288,7 @@
 #
 <Directory />
     Options FollowSymLinks
-    AllowOverride None
+    AllowOverride All
 </Directory>
 
 #
@@ -323,7 +323,7 @@
 # It can be "All", "None", or any combination of the keywords:
 #   Options FileInfo AuthConfig Limit
 #
-    AllowOverride None
+    AllowOverride All
 
 #
 # Controls who can get stuff from this server.
